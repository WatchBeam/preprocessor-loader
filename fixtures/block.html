<!-- @if PLATFORM == 'xbox' -->
<one-fish></one-fish>
<two-fish></two-fish>
<!-- @endif -->
<!-- @if PLATFORM == 'desktop' -->
<red-fish></red-fish>
<blue-fish></blue-fish>
<!-- @endif -->
<not-a-fish></not-a-fish>
