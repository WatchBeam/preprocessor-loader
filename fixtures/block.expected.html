<one-fish></one-fish>
<two-fish></two-fish>
<not-a-fish></not-a-fish>
